### ISSUE TITLE
Puzzle cannot be opened after import

### ISSUE THREAD

**Issue by philhanna** [philhanna/crossword#40]
Getting a key error in puzzle_screen() line 283:
```puzzle = Puzzle.from_json(session['puzzle'])```

**Comment by philhanna** [comment]
Figured out the problem.  The puzzle JSON is too big to be stored in the session:
```
/home/saspeh/.local/lib/python3.6/site-packages/werkzeug/wrappers.py:1116: UserWarning: The "b'session'" cookie is too large: the value was 4300 bytes but the header required 26 extra bytes. The final size was 4326 bytes but the limit is 4093 bytes. Browsers may silently ignore cookies larger than this.
  samesite=samesite
```

**Comment by philhanna** [comment]
Steps I took:
```bash
pip install Flask-Session
```

**Comment by philhanna** [comment]
Fixed with commit f8f6ea4
